lib/box/index.js

```diff
@@ -29,11 +30,14 @@ function Box(ctx, base, options) {
   this.watcher = null;
   this.Cache = ctx.model('Cache');
   this.File = this._createFileClass();
-  this.ignore = ctx.config.ignore;
+
+  let ignoreCfg = ignore;
   if (ctx.config.ignore) {
-    const targets = Array.isArray(ctx.config.ignore) ? ctx.config.ignore : [ctx.config.ignore];
-    this.options.ignored = (this.options.ignored || []).concat(targets.map(s => toRegExp(ctx, s)).filter(x => x));
+    if (ctx.config.ignore.length) ignoreCfg = ignoreCfg.concat(ctx.config.ignore);
```

Comment:
can't use `Array.push()`, don't know why :confused:

Reply:
https://mdn.io/push

When merging 2 arrays, `Array.prototype.push.apply(ignoreCfg, ctx.config.ignore);` should be used.

Spread syntax should be much more simpler: `ignoreCfg.push(...ctx.config.ignore)`

Reply:
https://dev.to/johncip/comment/aj26

Find a tip.

Reply:
After adding `console.log(ctx.config.ignore, ignoreCfg);` in my local machine, I got something like:

```

> hexo@4.1.0 test /mnt/d/Project/Sukka/hexo
> mocha test/index.js

[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]


  Hexo
    Box
      Box
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
[ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ] [ '**/themes/*/node_modules/**', '**/themes/*/.git/**' ]
        ✓ constructor - add trailing "/" to the base path
```

Reply:
oh well, let's use `concat()` then. I already reverted.